WIP: [GOV]: Update of govenance docs (#2539)
cntt-n/CNTT: doc/gov/chapters/chapter01.md

<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>

## Table of Contents
* [1.1 History of Anuket reference specifications](#1.1)
* [1.2 How to participate](#1.2)

<a name="1.1"></a>
## 1.1 History of Anuket reference specifications

> Anuket reference specifications were founded as the Cloud iNfrastructure Telco Task Force (CNTT) by AT&T, Vodafone, Verizon, Deutsche Telekom, Orange, and China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to join the Task Force. CNTT reached its first major milestone when it gained sponsorship and support of the GSMA and Linux Foundation Networking in Summer 2019. As of June 2020, there were over thirty operators and partners (VNF suppliers, third-party integrators, hw/sw suppliers) in its member commmunity and these numbers continue to grow. Together the CNTT community is leading the industry in creating a common infrastructure reference platform in the form of reference model and reference architecture definitions to better support Network Function Virtualization (NFV) applications for the Telecom industry as a whole. The working group was collaborating with OPNFV and other open source communities to operationalize and support reference implementations and reference certification platforms. 

Reviewer: The first sentence is very awkward "..specifications were founded ..."
```suggestion
The Cloud iNfrastructure Telco Task Force (CNTT) was founded by AT&T, Vodafone, Verizon, Deutsche Telekom, Orange, and China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to join the Task Force. CNTT reached its first major milestone when it gained sponsorship and support of the GSMA and Linux Foundation Networking in Summer 2019. As of June 2020, there were over thirty operators and partners (VNF suppliers, third-party integrators, hw/sw suppliers) in its member community and these numbers have continued to grow. CNTT was collaborating very closely with OPNFV and there were dependencies and overlap between the work of the two communities.  Hence, in 2021, the CNTT and OPNFV merged to form the Anuket Project.

The Anuket project community is leading the industry in creating a common infrastructure reference platform in the form of reference model and reference architecture definitions to better support virtualised and containerised Network Functions for the Telecom industry as a whole. The ANuket community is collaborating with other open source communities to operationalize and support reference implementations and reference certification platforms. 
```

> In the beginning of 2021 CNTT and OPNFV merged under the name [Anuket](https://anuket.io/) to leverage the synergies between the two projects. Former CNTT specifications are referred as Anuket reference specifications. Anuket operates under one Technical Oversight Commitee (TSC) and common governing rules documented in the Anuket Charter and in the Anuket Project Operations and Guidelines document. 

Reviewer: Some of this is history and in a previous suggestion moved
For both the Charter and the Operations Guideline we should include the links

```suggestion
The Anuket project operates under a Technical Oversight Committee (TSC) and governing rules documented in the Anuket Charter and in the Anuket Project Operations and Guidelines document. 
```

Author: Project operations document is not yet approved.

<a name="1.2"></a>
## 1.2 How to participate

> Participating to Anuket does not need any formal membership, and nothing to sign except the CLA. Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you do automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy. 

Reviewer: CLA needs expansion as well as a link and the other practices need a link.
Some minor typo corrections below
```suggestion
Participating in Anuket does not need any formal membership, and nothing to sign except the CLA. Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you do automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy. 
```

Author: Do we have approved CLA-s?

<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>

## Table of Contents
* [1.1 History of Anuket reference specifications](#1.1)
* [1.2 How to participate](#1.2)

<a name="1.1"></a>
## 1.1 History of Anuket reference specifications

The Cloud iNfrastructure Telco Task Force (CNTT) was founded by AT&T, Vodafone, Verizon, Deutsche Telekom, Orange, and China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to join the Task Force. CNTT reached its first major milestone when it gained sponsorship and support of the GSMA and Linux Foundation Networking in Summer 2019. As of June 2020, there were over thirty operators and partners (VNF suppliers, third-party integrators, hw/sw suppliers) in its member community and these numbers have continued to grow. CNTT was collaborating very closely with OPNFV and there were dependencies and overlap between the work of the two communities. In the beginning of 2021 CNTT and OPNFV merged under the name [Anuket](https://anuket.io/) to leverage the synergies between the two projects.

> The Anuket project community is leading the industry in creating a common infrastructure reference platform in the form of reference model and reference architecture definitions to better support virtualised and containerised Network Functions for the Telecom industry as a whole. The Anuket community is collaborating with other open source communities to operationalize and support reference implementations and reference certification platforms. 

Reviewer 2: ```suggestion
The Anuket project community is leading the industry in creating a common infrastructure reference platform in the form of reference model and reference architecture definitions to better support virtualised and containerised Network Functions for the Telecom industry as a whole. The Anuket community includes many open source development projects to create and support reference implementations, and develop tests for reference certification platforms. 
```

> The Anuket project operates under a Technical Oversight Committee (TSC) and governing rules documented in the [Anuket Charter](https://anuket.io/wp-content/uploads/sites/119/2021/07/Anuket-Charter_073021.pdf) and in the Anuket Project Operations and Guidelines document. 

Reviewer 2: This looks good, @gkunz

<a name="1.2"></a>
## 1.2 How to participate

> Participating in Anuket does not need any formal membership, and nothing to sign except the [CLA](https://anuket.io/cla/). Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you do automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy. 

Reviewer 2: ```suggestion
Participating in Anuket does not need any formal membership, and nothing to sign except the [CLA](https://anuket.io/cla/). Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy. 
```

- [LFN Anti-Trust Policy](https://lfnetworking.org/Antitrust%20Slide.pdf) 
- [GSMA Anti-Trust Policy](https://github.com/cntt-n/CNTT/blob/master/Anti-Trust_Notice_GSMA_20190911.png) 
- [CNTT Terms of Reference (ToR) LFN | GSMA](https://wiki.lfnetworking.org/display/LN/Terms+of+Reference?src=contextnavpagetreemode)

>   - Note: If you work for someone other than yourself (self-employed), then you must ensure that your employer also agrees to the ToR before participating.

Reviewer 2: ```suggestion
  - Note: If you are not self-employed, then you must ensure that your employer also agrees to the ToR before participating.
```

- [Code of Conduct LFN](https://lfprojects.org/policies/code-of-conduct/)
- [Trademark Policy](https://lfprojects.org/policies/trademark-policy/)

> Recommended checklist for participants to participate the Anuket community.

Reviewer 3: nit
```suggestion
Recommended checklist for participating in the Anuket community:
```

  - Review all legal policies, terms, and conditions
  - Signup for an [LFN ID](https://sso.linuxfoundation.org/login/?state=g6Fo2SBEbDAzLVFFN18teXdLMWZlUlVmWlVrMFpwZElHaUFtTqN0aWTZIGxCNVo3NExGOVpxNzJGT25YRW5vSkxMTlAwTENaLVdSo2NpZNkgcFZ0QzZ2R2ZmMnFBd0lxOXZVWGlhdTZzN3h6OWR3UUI&client=pVtC6vGff2qAwIq9vUXiau6s7xz9dwQB&protocol=oauth2&audience=https:%2F%2Fapi-gw.platform.linuxfoundation.org%2F&scope=openid%20profile%20email&response_type=code&response_mode=query&nonce=VldNTTJZdzc1UX5HVzBFMFd0dXBzTlc1UXpyQnJpVFlzfnY5WHA3OTl6SQ%3D%3D&redirect_uri=https:%2F%2Fmyprofile.lfx.linuxfoundation.org&code_challenge=VzQLzce3RLtJjTo1QmtKvZ3hrYd6J5U1yyQ4FXVodZ4&code_challenge_method=S256&auth0Client=eyJuYW1lIjoiYXV0aDAtc3BhLWpzIiwidmVyc2lvbiI6IjEuMTMuMiJ9)
  - Signup for a [GitHub ID](https://github.com/signup?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home)

>   - [Find a sub-project](https://wiki.anuket.io/display/HOME/Projects)

Reviewer 2: Add GitLab ID sign-up here, too.

Author: Fixed in 2c969bd

- [Code of Conduct LFN](https://lfprojects.org/policies/code-of-conduct/)
- [Trademark Policy](https://lfprojects.org/policies/trademark-policy/)

Recommended checklist for participating in the Anuket community:
  - Review all legal policies, terms, and conditions
  - Signup for an [LFN ID](https://sso.linuxfoundation.org/login/?state=g6Fo2SBEbDAzLVFFN18teXdLMWZlUlVmWlVrMFpwZElHaUFtTqN0aWTZIGxCNVo3NExGOVpxNzJGT25YRW5vSkxMTlAwTENaLVdSo2NpZNkgcFZ0QzZ2R2ZmMnFBd0lxOXZVWGlhdTZzN3h6OWR3UUI&client=pVtC6vGff2qAwIq9vUXiau6s7xz9dwQB&protocol=oauth2&audience=https:%2F%2Fapi-gw.platform.linuxfoundation.org%2F&scope=openid%20profile%20email&response_type=code&response_mode=query&nonce=VldNTTJZdzc1UX5HVzBFMFd0dXBzTlc1UXpyQnJpVFlzfnY5WHA3OTl6SQ%3D%3D&redirect_uri=https:%2F%2Fmyprofile.lfx.linuxfoundation.org&code_challenge=VzQLzce3RLtJjTo1QmtKvZ3hrYd6J5U1yyQ4FXVodZ4&code_challenge_method=S256&auth0Client=eyJuYW1lIjoiYXV0aDAtc3BhLWpzIiwidmVyc2lvbiI6IjEuMTMuMiJ9)
  - Signup for a [GitHub ID](https://github.com/signup?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home)
  - Signup for a [GitLab ID](https://gitlab.com/users/sign_up)

>   - [Find a sub-project](https://wiki.anuket.io/display/HOME/Projects)

Reviewer: Can we be a bit more descriptive such as "Choose sub-projects and begin contributing; review the sub-project information and on how to participate including meetings" or ...

Author: Added in b6a13e7
